src/utils/ucc_sys.c

```diff
@@ -62,3 +62,38 @@ ucc_status_t ucc_sysv_free(void *addr)
 
     return UCC_OK;
 }
+
+/*
+ * If a certain system constant (name) is undefined on the underlying system
+ * the sysconf routine returns -1.  ucs_sysconf return the negative value
+ * a user and the user is responsible to define default value or abort.
+ *
+ * If an error occurs sysconf modified errno and ucs_sysconf aborts.
+ *
+ * Otherwise, a non-negative values is returned.
+ */
+static long ucc_sysconf(int name)
```

Comment:
why do we need this wrapper? looks like it does nothing

Reply:
removed